lib/ApiRequestor.php

```diff
@@ -129,7 +129,7 @@ public function handleErrorResponse($rbody, $rcode, $rheaders, $resp)
         if (!is_array($resp) || !isset($resp['error'])) {
             $msg = "Invalid response object from API: $rbody "
               . "(HTTP response code was $rcode)";
-            throw new Exception\UnexpectedValueException($msg, $rcode, $rbody, $resp, $rheaders);
+            throw new Exception\UnexpectedValueException($msg);
```

Comment:
Can you say more about this change? I looked in the class but didn't find much context